dev/com.ibm.ws.opentracing/resources/com/ibm/ws/opentracing/resources/Opentracing.nlsprops

```diff
@@ -66,26 +66,14 @@ OPENTRACING_COULD_NOT_CREATE_TRACER=CWMOT0001E: Invocation of user supplied Open
-OPENTRACING_NO_TRACER_FOR_INBOUND_REQUEST.useraction=Examine previous errors to determine why a Tracer is not available.
-
-OPENTRACING_NO_SPAN_FOR_RESPONSE_TO_INBOUND_REQUEST=CWMOT0003E: The Span created for an inbound request is not available for the response to the request. The inbound request will not be correlated the upstream service.
-OPENTRACING_NO_SPAN_FOR_RESPONSE_TO_INBOUND_REQUEST.explanation=The Span created when the request arrived is not available when the response returned. No Span information will be provided to the Opentracing service.
-OPENTRACING_NO_SPAN_FOR_RESPONSE_TO_INBOUND_REQUEST.useraction=Examine previous errors to determine why a Span is not available.
-
-OPENTRACING_NO_TRACER_FOR_OUTBOUND_REQUEST=CWMOT0004E: No Tracer was found for an outbound request. The outbound request will not be correlated with the inbound request.
-OPENTRACING_NO_TRACER_FOR_OUTBOUND_REQUEST.explanation=A Tracer implementation was not available when a request is sent from the application. This outbound request will not provide any information to the Opentracing service.
-OPENTRACING_NO_TRACER_FOR_OUTBOUND_REQUEST.useraction=Examine previous errors to determine why a Tracer is not available.
-
-OPENTRACING_NO_SPAN_FOR_RESPONSE_TO_OUTBOUND_REQUEST=CWMOT0005E: The Span created for an outbound request is not available for the response to the request. The outbound request will not be correlated with the inbound request.
-OPENTRACING_NO_SPAN_FOR_RESPONSE_TO_OUTBOUND_REQUEST.explanation=The Span created when the request was sent is not available when the response returned. No Span information will be provided to the Opentracing service.
-OPENTRACING_NO_SPAN_FOR_RESPONSE_TO_OUTBOUND_REQUEST.useraction=Examine previous errors to determine why a Span is not available.
-
 OPENTRACING_TRACERFACTORY_RETURNED_NULL=CWMOT0006E: Invocation of user supplied OpentracingTracerFactory.newInstance(...) method returned null
 OPENTRACING_TRACERFACTORY_RETURNED_NULL.explanation=The feature constructs Tracer implementations with a user provided OpentracingTracerFactory. The OpentracingTracerFactory.newInstance method returned null.
 OPENTRACING_TRACERFACTORY_RETURNED_NULL.useraction=Determine why the user supplied OpentracingTracerFactory newInstance method returned null.
 
+OPENTRACING_NO_TRACERFACTORY=CWMOT0008E: OpenTracing will not track requests because an OpentracingTracerFactory has not been supplied.
+OPENTRACING_NO_TRACERFACTORY.explanation=The OpenTracing features will not function properly because an OpentracingTracerFactory is not provided. JAX-RS requests will execute normally, but they will not be tracked.
+OPENTRACING_NO_TRACERFACTORY.useraction=Consult the documentation on how to enable distributed tracing with the OpenTracing feature.
```

Comment:
ID review:

OPENTRACING_NO_TRACERFACTORY=CWMOT0008E: OpenTracing cannot track requests because an OpentracingTracerFactory was not provided.
**What's an OpentracingTracerFactory? A class/method/object/etc.?**
OPENTRACING_NO_TRACERFACTORY.explanation=The OpenTracing features will not function properly because an OpentracingTracerFactory is not provided. JAX-RS requests will execute normally, but they will not be tracked.
OPENTRACING_NO_TRACERFACTORY.useraction=See the documentation about how to enable distributed tracing with the OpenTracing feature.
**Is the "feature" here the same feature as the "features" mentioned in the explanation, or is it meant as in Liberty feature? Consider clarifying the terminology.**

Reply:
@cjcarpen Thanks, I've clarified that OpentracingTracerFactory is a class. I also removed the discussion about features in useraction as really what we want the user to do is to search the documentation for "enable distributed tracing" so I've made the sentence end with that.